tests/test_api.rs

```diff
@@ -4778,3 +4778,102 @@ fn icu_collator() {
   let script = v8::Script::compile(scope, source, None).unwrap();
   assert!(script.run(scope).is_some());
 }
+
+fn create_module<'s>(
+  scope: &mut v8::HandleScope<'s, v8::Context>,
+  source: &str,
+  code_cache: Option<v8::UniqueRef<v8::CachedData>>,
+) -> v8::Local<'s, v8::Module> {
+  let source = v8::String::new(scope, source).unwrap();
+  let resource_name = v8::String::new(scope, "<resource>").unwrap();
+  let source_map_url = v8::undefined(scope);
+  let script_origin = v8::ScriptOrigin::new(
+    scope,
+    resource_name.into(),
+    0,
+    0,
+    false,
+    0,
+    source_map_url.into(),
+    false,
+    false,
+    true,
+  );
+  let source = match code_cache {
+    Some(x) => v8::script_compiler::Source::new_with_cached_data(
+      source,
+      &script_origin,
+      x,
+    ),
+    None => v8::script_compiler::Source::new(source, &script_origin),
+  };
+  let module = v8::script_compiler::compile_module(scope, source).unwrap();
+  module
+}
+
+fn create_unbound_module_script<'s>(
+  scope: &mut v8::HandleScope<'s, v8::Context>,
+  source: &str,
+  code_cache: Option<v8::UniqueRef<v8::CachedData>>,
+) -> v8::Local<'s, v8::UnboundModuleScript> {
+  let module = create_module(scope, source, code_cache);
+  module.get_unbound_module_script(scope)
+}
+
+#[test]
+fn unbound_module_script_conversion() {
+  let _setup_guard = setup();
+  let isolate = &mut v8::Isolate::new(Default::default());
+  let scope = &mut v8::HandleScope::new(isolate);
+  let context = v8::Context::new(scope);
+  let mut scope = v8::ContextScope::new(scope, context);
+  create_unbound_module_script(&mut scope, "'Hello ' + value", None);
+}
```

Comment:
This test has no asserts. Seems kind of useless?

Reply:
`UnboundModuleScript` doesn't have any methods except `create_code_cache` so this test just makes sure that none of the `.unwrap()`s fail.